pkg/flowkit/values.go

```diff
@@ -22,21 +22,24 @@ import (
 	"github.com/onflow/cadence"
 )
 
-func NewStakingInfoFromValue(value cadence.Value) map[string]interface{} {
-	stakingInfo := make(map[string]interface{})
+func NewStakingInfoFromValue(value cadence.Value) []map[string]interface{} {
+	stakingInfo := make([]map[string]interface{}, 0)
 	arrayValue := value.(cadence.Array)
```

Comment:
Is this guaranteed to be array else should you do safe type assertion with a check `v, ok := value.(cadence.Array)` to avoid panic.